modules/apps/frontend-js/frontend-js-aui-web/src/main/resources/META-INF/resources/liferay/form.js

```diff
@@ -25,6 +25,8 @@ AUI.add(
 
 		var TABS_SECTION_STR = 'TabsSection';
 
+		var REGEX_EMAIL = /^[\w!#$%&'*+/=?^_`{|}~-]+(?:\.[\w!#$%&'*+/=?^_`{|}~-]+)*@(?:\w(?:[\w-]*\w)?\.)+(\w(?:[\w-]*\w))$/;
```

Comment:
This overrides de [AUI regex for email](https://github.com/liferay/alloy-ui/blob/master/src/aui-form-validator/js/aui-form-validator.js#L73). While it might be exactly what we want, it can also be confusing.

@jonmak08, what do you think? Are we good moving this here so we don't have to cut a new AlloyUI release and this only affects DXP instances that backport this specific change?

Reply:
that looks good and makes sense to me.  I think we can proceed.